src/shared/codelens/localLambdaRunner.ts

```diff
@@ -358,6 +367,10 @@ export async function makeInputTemplate(params: {
         }
     }
 
+    if (params.globals) {
+        newTemplate.withGlobals(params.globals)
```

Comment:
I missed this in the first two reviews: This depends on the implementation detail that `withGlobals` returns `this`. To protect against implementation changes, suggest to use `newTemplate = newTemplate.withGlobals(...)`. Same comment for `withEnvironment` and `withMemorySize`.